googlemock/include/gmock/gmock-generated-function-mockers.h

```diff
@@ -320,6 +320,15 @@ class FunctionMocker<R(A1, A2, A3, A4, A5, A6, A7, A8, A9, A10)> : public
   }
 };
 
+
+template<typename T> T& RemoveVolatileCast(T & v) {
```

Comment:
Consistency:
  `(T& v)`, not `(T & v)`
  `template <typename T>`, not `template<typename T>`.